pkgs/top-level/all-packages.nix

```diff
@@ -4016,6 +4016,8 @@ in
 
   ugarit-manifest-maker = callPackage ../tools/backup/ugarit-manifest-maker { };
 
+  unar = callPackage ../tools/archivers/unar { };
```

Comment:
k. Though it should compile with gcc if we were to compile it with objc support using ```( gcc.cc.override { langObjC = true; } )``` in buildInputs. I just haven't tested it. ;)